src/commitment_scheme/kzg10/key.rs

```diff
@@ -7,7 +7,10 @@
 //! Key module contains the utilities and data structures
 //! that support the generation and usage of Commit and
 //! Opening keys.
-use super::{AggregateProof, Commitment, Proof};
+
+#[cfg(test)]
```

Comment:
I'm not a big fan of having testing code and actual code mixed together. This should be moved on the unit test (`mod test` section), as well as any code in this scope that is using that and / or is solely made for testing.

I'm open to discuss if there is any advantages in having such "spaghetti code", since from my prospective there isn't, so unless there is something I'm not seeing, please ensure the test code and the actual code are properly separated.